libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/infopanel/POINameComponent.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.navigation.dropin.component.infopanel
+
+import androidx.appcompat.widget.AppCompatTextView
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.ui.app.internal.Store
+import com.mapbox.navigation.ui.base.lifecycle.UIComponent
+
+@ExperimentalPreviewMapboxNavigationAPI
+internal class POINameComponent(
+    private val store: Store,
+    private val textView: AppCompatTextView,
```

Comment:
Let us also inject the ability to change the appearance of `textView`
`private val textAppearance: StateFlow<Int>`